robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Open Test Browser
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Test Cases ***
+
+Verify Error Message with GE Disabled and enable GE
+    [Documentation]                           This test case checks if advanced mapping is enabled and if disabled, enables it. 
+    ...                                       Checks if gift entry is disabled and if enabled, disables it and verifies that GE page has error. 
+    ...                                       Then enables gift entry and veries that gift entry page lands on templates page and has default template  
```

Comment:
Small correction! `verifies` is misspelled :)

Reply:
thanks will correct it